sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallConnectionAsync.java

```diff
@@ -0,0 +1,442 @@
+    public Mono<Void> addParticipant(
+        CommunicationIdentifier participant,
+        String alternateCallerId,
+        String operationContext) {
+        try {
+            Objects.requireNonNull(participant, "'participant' cannot be null.");
+            InviteParticipantsRequest request = InviteParticipantRequestConverter.convert(participant,
+                alternateCallerId,
+                operationContext,
+                null);
+            return callConnectionInternal.inviteParticipantsAsync(callConnectionId, request)
+                .onErrorMap(CommunicationErrorException.class, CallingServerErrorConverter::translateException);
+        } catch (RuntimeException ex) {
+            return monoError(logger, ex);
+        }
+    }
+
+    /**
+     * Add a participant to the call.
+     *
+     * @param participant Invited participant.
+     * @param alternateCallerId The phone number to use when adding a phone number participant.
+     * @param operationContext The value to identify context of the operation. This is used to co-relate other
+     *                         communications related to this operation
+     * @return response for a successful add participant request.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<Void>> addParticipantWithResponse(
+        CommunicationIdentifier participant,
+        String alternateCallerId,
+        String operationContext) {
+        return addParticipantWithResponse(participant, alternateCallerId, operationContext, Context.NONE);
+    }
+
+    Mono<Response<Void>> addParticipantWithResponse(
+        CommunicationIdentifier participant,
+        String alternateCallerId,
+        String operationContext,
+        Context context) {
+        try {
+            Objects.requireNonNull(participant, "'participant' cannot be null.");
+            InviteParticipantsRequest request =
+                InviteParticipantRequestConverter
+                    .convert(participant, alternateCallerId, operationContext, null);
+            return withContext(contextValue -> {
+                contextValue = context == null ? contextValue : context;
+                return callConnectionInternal
+                    .inviteParticipantsWithResponseAsync(callConnectionId, request, contextValue)
+                    .onErrorMap(CommunicationErrorException.class, CallingServerErrorConverter::translateException);
+            });
+        } catch (RuntimeException ex) {
+            return monoError(logger, ex);
+        }
+    }
+
+    /**
+     * Remove a participant from the call.
+     *
+     * @param participantId Participant id.
+     * @return response for a successful remove participant request.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Void> removeParticipant(String participantId) {
+        try {
+            Objects.requireNonNull(participantId, "'participantId' cannot be null.");
```

Comment:
Participant ID is a server side param right? In this case, we do not need to check on client side according to Azure guidelines.

Reply:
null check for server side params removed here and other places